src/test/cypress/e2e/exercises/file-upload/FileUploadExerciseAssessment.cy.ts

```diff
@@ -0,0 +1,80 @@
+import { FileUploadExercise } from 'app/entities/file-upload-exercise.model';
+import { Course } from 'app/entities/course.model';
+import {
+    courseAssessment,
+    courseManagement,
+    courseManagementRequest,
+    courseOverview,
+    exerciseAssessment,
+    exerciseResult,
+    fileUploadExerciseAssessment,
+    fileUploadExerciseEditor,
+    fileUploadExerciseFeedback,
+} from 'src/test/cypress/support/artemis';
+import { admin, instructor, studentOne, tutor } from 'src/test/cypress/support/users';
+import { convertModelAfterMultiPart } from 'src/test/cypress/support/requests/CourseManagementRequests';
+
+// Common primitives
+const tutorFeedback = 'Try to use some newlines next time!';
+const tutorFeedbackPoints = 4;
+const complaint = "That feedback wasn't very useful!";
+
+describe('File upload exercise assessment', () => {
+    let course: Course;
+    let exercise: FileUploadExercise;
+
+    before('Creates a file upload exercise and makes a student submission', () => {
+        cy.login(admin);
+        courseManagementRequest.createCourse().then((response) => {
```

Comment:
Suggestion for potential future PR: rewrite using await (everywhere)

Reply:
Good point, will look into it